packages/typespec-python/scripts/eng/regenerate.ts

```diff
@@ -236,6 +238,9 @@ function addOptions(spec: string, generatedFolder: string, flags: RegenerateFlag
             options["company-name"] = "Unbranded";
         }
         options["examples-dir"] = toPosix(join(dirname(spec), "examples"));
+        if (options["enable-typespec-namespace"] === undefined) {
+            options["enable-typespec-namespace"] = "false";
+        }
```

Comment:
shall we do the default in emitter cause we decided to always set it to false for azure for now.

Reply:
we default it as true for unbranded which will cause test change. So we always set it as false. After this PR merged, I will update test of unbranded then we can remove this configuration.